lua/lspconfig/server_configurations/foam.lua

```diff
@@ -0,0 +1,26 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    -- `foam-ls` can be a shell script executing:
+    -- node /path/to/foam-language-server/lib/foam-ls.js --stdio
+    cmd = { 'foam-ls' },
+    filetypes = { 'foam', 'OpenFOAM' },
+    root_dir = util.root_pattern('system/controlDict'),
+    log_level = vim.lsp.protocol.MessageType.Warning,
+  },
+  docs = {
+    package_json = 'https://raw.githubusercontent.com/FoamScience/foam-language-server/master/package.json',
```

Comment:
This `package_json` is currently only used to pull the server's configuration schema when generating the `server_configurations.md` docs. It seems like this package.json does not provide any configuration schema, so I think this should be removed.

```suggestion

```